mono/mini/il2cpp-stubs.cpp

```diff
@@ -224,8 +224,7 @@ const char* il2cpp_mono_field_get_name (Il2CppMonoClassField *field)
 
 Il2CppMonoMethod* il2cpp_mono_class_get_method_from_name (Il2CppMonoClass *klass, const char *name, int param_count)
```

Comment:
Could we replace this stub with a direct call to `il2cpp_class_get_method_from_name` on the IL2CPP embedding API?